api.ts

```diff
@@ -1127,6 +1128,23 @@ export interface SyncFormulaDef<
    * @hidden
    */
   updateOptions?: Pick<CommonPackFormulaDef<ParamDefsT>, 'extraOAuthScopes'>;
+
+  /**
+   * Given a set of rows, return permission lists, keyed by the row's idProperty value.
+   *
+   * TODO(patrick): Unhide this
+   * @hidden
+   */
+  getPermissions?(rows: Array<ObjectSchemaDefinitionType<K, L, SchemaT>>): Promise<Record<string, Permission[]>>;
```

Comment:
Nice love this. Only thing to think through is if this could take in just the id field or if it needs the entire row? Just want to think about _when_ we will be syncing permissions. Like if we want to sync them on a different cadence than the rows will we still have access to the entire row content?